example_13/doc/userdoc.rst

```diff
@@ -0,0 +1,471 @@
+    threedofbot_joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster] active
+    threedofbot_pid_gain_controller[forward_command_controller/ForwardCommandController] active
+    threedofbot_position_controller[forward_command_controller/ForwardCommandController] active
+
+
+6. Deactivate ``RRBotSystemPositionOnly`` and its position controller (first). Call
+
+  .. code-block:: shell
+
+    ros2 control switch_controllers --deactivate rrbot_position_controller
+    ros2 control set_hardware_component_state RRBotSystemPositionOnly inactive
+
+  Scenario state:
+
+  - right robot is now "standing" at the last position
+  - left robot is moving
+  - middle robot is moving
+
+  Hardware status: ``RRBotSystemPositionOnly`` is in inactive state.
+
+  .. code-block:: shell
+
+    $ ros2 control list_hardware_components
+    ...
+    Hardware Component 4
+        name: RRBotSystemPositionOnly
+        type: system
+        plugin name: ros2_control_demo_example_5/RRBotSystemPositionOnlyHardware
+        state: id=2 label=inactive
+        command interfaces
+                rrbot_joint1/position [available] [unclaimed]
+                rrbot_joint2/position [available] [unclaimed]
+    ...
+
+  Controllers status: ``rrbot_position_controller`` is now in inactive state
+
+  .. code-block:: shell
+
+    $ ros2 control list_controllers
+    joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster] active
+    rrbot_external_fts_broadcaster[force_torque_sensor_broadcaster/ForceTorqueSensorBroadcaster] active
+    rrbot_joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster] active
+    rrbot_position_controller[forward_command_controller/ForwardCommandController] inactive
+    rrbot_with_sensor_fts_broadcaster[force_torque_sensor_broadcaster/ForceTorqueSensorBroadcaster] active
+    rrbot_with_sensor_joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster] active
+    rrbot_with_sensor_position_controller[forward_command_controller/ForwardCommandController] active
+    threedofbot_joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster] active
+    threedofbot_pid_gain_controller[forward_command_controller/ForwardCommandController] active
+    threedofbot_position_controller[forward_command_controller/ForwardCommandController] active
+
+
+7. Set ``RRBotSystemPositionOnly`` in unconfigured state, and deactivate its joint state broadcaster. Also restart global joint state broadcaster. Call
+
+  .. code-block:: shell
+
+    ros2 control switch_controllers --deactivate rrbot_joint_state_broadcaster joint_state_broadcaster
+    ros2 control set_hardware_component_state RRBotSystemPositionOnly unconfigured
+    ros2 control switch_controllers --activate joint_state_broadcaster
+
+  Scenario state (everything is broken during ``joint_state_broadcaster`` restart):
+
+  - right robot is standing still.
+  - left robot is moving
+  - middle robot is moving
+
```

Comment:
I could reproduce it and it is working

The only thing I'm not sure if it makes sense if after unconfiguring the HW interface, how the joint_state_broadcaster still able to publish the state of the right robot.


Technically, it doesn't make sense

Reply:
Fixes it properly : https://github.com/ros-controls/ros2_controllers/pull/1331

Reply:
Thanks, I have to test that once more.